<dec f='llvm/llvm/include/llvm/DebugInfo/PDB/GenericError.h' l='36' type='const std::error_category &amp; llvm::pdb::PDBErrCategory()'/>
<use f='llvm/llvm/include/llvm/DebugInfo/PDB/GenericError.h' l='39' u='c' c='_ZN4llvm3pdb15make_error_codeENS0_14pdb_error_codeE'/>
<def f='llvm/llvm/lib/DebugInfo/PDB/GenericError.cpp' l='46' type='const std::error_category &amp; llvm::pdb::PDBErrCategory()'/>
